### ISSUE TITLE
Make examples import from package names

### ISSUE THREAD

**Issue by mjackson** [reach/reach-ui#294]
## 📖 Documentation

Our examples should `import` from the bare package name instead of from a relative `src` directory. This will let people copy/paste our example code directly into their app more easily.

```js
// instead of this
import { Slider, SliderHandle } from '../../slider/src';
// do this
import { Slider, SliderHandle } from '@reach/slider';
```

Important: _This doesn't mean that we have to build everything in dev_, just that we are changing the `import` paths. We can do some translation in our build tool to translate `@reach/slider` to `packages/slider/src` so we can still use the source code directly when running the examples.

**Comment by jescowuester** [comment]
I think [babel plugin transform imports](https://www.npmjs.com/package/babel-plugin-transform-imports) is probably the easiest way to make this work smoothly between rollup and Webpack.

**Comment by ernestognw** [comment]
Hi @mjackson, I want to take this challenge, but I can't find where are the relative `src` relative directories to change. 

Can u provide me more info? Thanks!

**Comment by jescowuester** [comment]
@ernestognw every package has an `examples` folder and a `src` folder. The only way I see this beeing done is to manually set an alias in the storybook webpack config for every included package since things like @reach/router aren't in the repo.
```
resolve: {
  alias: {
    "@reach/alert": path.resolve(__dirname, "../packages/alert"),
	"@reach/alert-dialog": path.resolve(__dirname, "../packages/alert-dialog"),
    ...etc
  }
}
```

**Comment by syncatallah** [comment]
Solved #300
